sympy/stats/frv_types.py

```diff
@@ -88,7 +106,7 @@ def dict(self):
 
     @property
     def set(self):
-        return self.args
+        return list(self.args)
```

Comment:
why `list`?

Reply:
In `frv_types`, the distributions were returning the set as `list`, so to keep everything in agreement, I made this change.